### ISSUE TITLE
Playlist only available for the user

### ISSUE THREAD

**Issue by Trantium** [flyth/ts3soundbot#139]
Hi!

Is it possible to make playlists created by User A only visible for him, add his own songs and such? Now the playlists and all the songs are visible by all users across all instances. And approve a user to only control that instance instead of all available ones.

See it as a linux virtualisation with containers for example.

**Comment by flyth** [comment]
Yup, that's something I'll look into sooner or later :) Thanks for suggesting it!
